executor/builder.go

```diff
@@ -195,6 +197,12 @@ func (b *executorBuilder) buildCancelDDLJobs(v *plannercore.CancelDDLJobs) Execu
 	}
 	return e
 }
+func (b *executorBuilder) buildChange(v *plannercore.Change) Executor {
```

Comment:
Add a blank line in front of this function.

Reply:
finished